map-widget/qml/MapWidget.qml

```diff
@@ -9,13 +9,10 @@ Item {
 	property alias mapHelper: mapHelper
 	property alias map: map
 
-	signal selectedDivesChanged(var list)
-
 	MapWidgetHelper {
 		id: mapHelper
 		map: map
 		editMode: false
-		onSelectedDivesChanged: rootItem.selectedDivesChanged(list)
```

Comment:
so this field here acts like a slot for the selectedDivesChanged signal that MapWidgetHelper has and it does re-emit the signal.

i added it in https://github.com/Subsurface-divelog/subsurface/commit/8b43633c7d22a31f882919f535a8fc332f61af62 but i don't remember why exactly - i.e. GUI wise.
it was definitely solving something, yet the component saw a number of refactors.

the addition of this connection https://github.com/Subsurface-divelog/subsurface/commit/48b8129137ed3db1c4b0940301c8bfc9662bcee3#diff-3a4ab556816851de53b67cbf136cc2aaR51
in my opinion made the above line redundant and should have been removed back then.